### PR TITLE
[FIX] web_notify README

### PR DESCRIPTION
**Pull request by aitorbouzas**
@simahawk The readme contained some old instructions

### CODE REVIEW

File: web_notify/README.rst [hunk 1]
```diff
@@ -64,7 +64,7 @@ or
    :alt: Sample notifications
 
 You can test the behaviour of the notifications by installing this module in a demo database.
-   Click in your user logo in the top right corner of the screen then click "Preferences".
```
Comment (simahawk) [inline web_notify/README.rst:67]: this is duplicated below, keep instructions in USAGE only

**Review by simahawk — CHANGES_REQUESTED** [review]
also, pls, try to include module name in PR title and commit message ;)

File: web_notify/README.rst (hunk 1 above)
Comment (aitorbouzas) [reply web_notify/README.rst:67]: What's duplicated? README.rst is the result of the new generator that takes content from ```readme/``` folder...

Comment (simahawk) [reply web_notify/README.rst:67]: hmmm sorry, I mis-read the path above the content! :stuck_out_tongue_closed_eyes:

**Review by simahawk — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**